### ISSUE TITLE
I Would pay £ for facebook plugin 

### ISSUE THREAD

**Issue by chronic0383** [Athlon1600/php-proxy-app#88]
I'm really enjoying using this proxy software, and i would pay for development of a facebook plugin, that could be shared with all users ?

**Comment by ash121121** [comment]
What's your budget

**Comment by chronic0383** [comment]
Humm i don't know , how much are you looking for? and how long would it take to make ?

**Comment by chronic0383** [comment]
@ash121121 Any thoughts on prices buddy ?

**Comment by Athlon1600** [comment]
Making such plugin would be extremely difficult given how much facebook depends on javascript and the fact that it blocks account access from "unfamiliar" IP addresses. Facebook would not by default allow someone from california to login to an account that was first created in new york. I'm not even aware of any web proxies that can bypass any of these problems. I could be wrong though.  proxfree I know does not support facebook.

**Comment by chronic0383** [comment]
Agreed I know this will be a hard plugin to make, gylpe had a plugin in the past. I know a site that is using PHP proxy that seems to have Facebook working, just unsure how they have written the plugin 😞.

**Comment by pluriglot** [comment]
Hi,

1.) Issue on loading the Facebook videos:---
I think it is possible for PHP-Proxy to load the facebook videos, because I found two proxy sites powered by PHP-Proxy, as follows:
https://pi.dvbris.com/public/proxy/index.php
http://qeentu.com/proxy/
The above two websites are able to encrypt the facebook video's URLs, therefore, they can load the facebook videos.



2.) Issue on signing in  to Facebook account:---
I know a nice website:--- https://whoer.net/webproxy
This web proxy (https://whoer.net/webproxy) can allow me to log in to my Facebook account. I was successfully logged in to my Facebook account. Please check the below attachment.

![whoer](https://user-images.githubusercontent.com/34707208/36984433-da880e32-20cf-11e8-8ed0-174ecdc0a780.png)

It looks like  this site (https://whoer.net/webproxy) is not powered by PHP-Proxy. It is different from PHP-Proxy.




I would appreciate it very much if you could solve any one of the above two issues, either the issue with loading the facebook videos or the issue with logging in to facebook account. And also, I will try to find out more sites which support Facebook Login.

**Comment by pluriglot** [comment]
I found a group of proxy sites which support Facebook login. Some of them are listed below:---
https://polaris.cx/index.php
https://helloworld.cx/index.php
https://firefly.cx/index.php
https://cyberbay.org/index.php
https://freedom.cx/index.php

I think the above proxy sites are written in PHP, but I am not sure whether they are powered by Glype.

Please also check the below links to see a full list of the similar websites:---
http://unicorn.cx/p/anonymous-proxy-sites
https://firefly.cx/p/school-proxy
https://cyberbay.org/p/ssl-proxy-list
http://freedom.cx/p/ssl-proxy-sites

Let me take one of these sites for an example:--- https://polaris.cx

I was successfully logged in to my Facebook account through the above proxy site (https://polaris.cx). Please check the below screenshot.
![polaris](https://user-images.githubusercontent.com/34707208/37101100-b0379454-225f-11e8-99cf-b278c8ca44f5.png)



I was using the above-mentioned website on my desktop computer. But the interface of my Facebook account was automatically changed into the mobile version, instead of desktop version.

**Comment by zxq2233** [comment]
Hi, friend!
This webproxy site:
http://hesti.net/
Support facebook login

@Athlon1600 

just unsure how they have written the plugin

**Comment by ash121121** [comment]
I will take s look myself in the next few days and post here if I manage to fix if

**Comment by Benji-Collins** [comment]
On the whole 'other proxies that work with facebook' thing, CGIProxy used to work, not sure if it still does. I think that delving in to the proxies that @pluriglot suggested might be helpful.

**Comment by wapfalls** [comment]
This proxy site (https://www.crocomania.network) almost unlocks all of the sites i tried. Not sure what script is being used. Any idea?
